src/App.tsx

```diff
@@ -1,7 +1,37 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    key: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.ListenerTypeKey);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.ListenerTypeKey);
+  }
+
+  ListenerTypeKey = (event: KeyboardEvent) => {
```

Comment:
keep naming rules 
![image](https://user-images.githubusercontent.com/122989585/219870473-3a40b1cd-367a-4ee6-9eb6-f204dba423d1.png)

```suggestion
  handleKeyPressed= (event: KeyboardEvent) => {
```